src/main/java/hudson/plugins/analysis/collector/AnalysisPublisher.java

```diff
@@ -149,6 +68,21 @@ public boolean isCheckStyleActivated() {
     }
 
     /**
+     * Needed by workflow snippet generator.
+     * Having this ugly getter naming to keep backward configuration file compatibility on fields names.
+     * 
+     * @see #isCheckStyleActivated()
+     */
+    public boolean isIsCheckStyleActivated() {
```

Comment:
Well if you change it later you need to keep the ugly getters for backwards compat anyways :) But I have no opinion on it, just a tip.